src/eval-service.js

```diff
@@ -9,11 +7,13 @@ class EvalService {
             $event: options.$event
         };
 
-        if(options.attrs) {
+        if (options.attrs) {
             locals.attrs = options.attrs;
         }
 
-        scope.$eval(options.fn, locals);
+        scope.$apply(() => {
+            scope.$eval(options.fn, locals);
```

Comment:
Okay cool, that makes sense. Just need to make sure the old function way works, which it should, but just have to say it for the record!